lib/config.js

```diff
@@ -381,6 +381,11 @@ Config.prototype.buildProjects = function () {
       arg_name: projectName.replace('-', '_')
     }
   })
+
+  // TODO(bridiver) - we don't really need this projects stuff anymore because
+  // we only have two now (chrome and brave-core) and anything else will be
+  // managed by deps
+  this.projects['chrome'].url = this.chromiumRepo
```

Comment:
is it `.repository.url`?

Reply:
not at this point, see the code above that is collapsed